cases-SEP24/sep10.test.js

```diff
@@ -315,14 +325,14 @@ describe("SEP10", () => {
     it("succeeds for a signer without an account", async () => {
       const kp = StellarSDK.Keypair.random();
       const { token, logs } = await getSep10Token(url, kp, [kp]);
-      expect(token).toBeTruthy();
+      expect(token).signersAssertion(logs, [kp], accountPool);
```

Comment:
I don't think this is really asserting anything about signers. Can we change the name to `orElseLogKeypairs()`?

Reply:
I can get behind that name. 👍